examples/default/Makefile

```diff
@@ -4,6 +4,8 @@ APPLICATION = default
 # If no BOARD is found in the environment, use this default:
 BOARD ?= native
 
+BOARD_INSUFFICIENT_MEMORY := nucleo32-f031
```

Comment:
indeed, I don't remember why I added it there. Change reverted, I'll squash/rebase if CI passes.